webpack.mix.js

```diff
@@ -6,12 +6,16 @@ const mix = require('laravel-mix');
  |--------------------------------------------------------------------------
  |
  | Mix provides a clean, fluent API for defining some Webpack build steps
- | for your Laravel applications. By default, we are compiling the CSS
+ | for your Laravel application. By default, we are compiling the Sass
  | file for the application as well as bundling up all the JS files.
  |
  */
 
 mix.js('resources/js/app.js', 'public/js')
-    .postCss('resources/css/app.css', 'public/css', [
-        //
-    ]);
+    .js('resources/js/styleguide.js', 'public/js')
+    .sass('resources/sass/app.scss', 'public/css')
+    .sass('resources/sass/styleguide.scss', 'public/css');
+
+if (!mix.inProduction()) {
+    mix.sourceMaps();
+}
```

Comment:
I prefer to use versioning in production to avoid displaying "cached" styles/scripts

```suggestion
}

if (mix.inProduction()) {
    mix.version()
        .polyfill() // Support IE 11
}

```

Reply:
You are totally right. Updating..